TalkAloud/TalkAloud/Model/CoreDataManager.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  CoreDataManager.swift
+//  TalkAloud
+//
+//  Created by Justin Bengtson on 6/25/20.
+//  Copyright © 2020 Justin Bengtson. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+class CoreDataManager: NSManagedObject {
```

Comment:
Why is your CoreDataManager a subclass of NSManagedObject?

Reply:
I was thinking at the time I was gonna need methods from NSManagedObject but importing from CoreData provides those for me so I shouldn't need it.